doc/whats_new/v0.20.rst

```diff
@@ -108,6 +107,11 @@ Decomposition, manifold learning and clustering
 - Fixed a bug in :func:`datasets.fetch_kddcup99`, where data were not properly
   shuffled. :issue:`9731` by `Nicolas Goix`_.
 
+Model evaluation and meta-estimators
```

Comment:
This is not the right subsection, there should be one with "metrics" in it, look in older version whats_new if you can not find it in this file.